File: Java Programing Basics/ExamTraning/src/Cake.java
import java.util.Scanner;

public class Cake {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        int a = Integer.parseInt(scanner.nextLine());
        int b = Integer.parseInt(scanner.nextLine());

        int area = a * b ;
        int count = 0;

        for (; count < area ; ){
            String peaces = scanner.nextLine();
            if (peaces.equals("STOP")){
                break;
            }
            int ioo = Integer.parseInt(peaces);
            count += ioo;
        }
        int diff = area - count;
        if (count > area){
            System.out.printf("No more cake left! You need %d pieces more.", Math.abs(diff));
        }else
            System.out.printf("%d pieces are left.", diff);

    }
}

File: Python/While_Loop(EXERCISE)/Cake.py
width = int(input())
lenght = int(input())
pieces_people_get = ""
total_pieces = 0

area = width * lenght

while total_pieces <= area:
    pieces_people_get = input()

    if pieces_people_get == "STOP":
        deff = area - total_pieces
        print(f"{deff} pieces are left.")
        break
    else:
        pieces_people_get = int(pieces_people_get)
        total_pieces += pieces_people_get

if total_pieces > area:
    deff = total_pieces - area
    print(f"No more cake left! You need {deff} pieces more.")
